_posts/2014-03-13-living-with-salespeople.md

```diff
@@ -1,4 +1,6 @@
 ---
+redirect_from:
```

Comment:
All files below this point are just the redirects from old url to new url as per jekyll-redirect-from gem.